### PR TITLE
images: add wireguard-tools to Debian images

### PR DESCRIPTION
**Pull request by subhoghoshX**
* [x] image-refresh debian-stable
 * [x] image-refresh debian-testing
 * [x] image-refresh ubuntu-stable
 * [x] image-refresh ubuntu-2204

### CODE REVIEW

File: images/scripts/debian.setup [hunk 1]
```diff
@@ -43,6 +43,7 @@ tuned \
 xdg-utils \
 udisks2 \
 udisks2-lvm2 \
+wireguard-tools \
```
Comment (martinpitt) [inline images/scripts/debian.setup:46]: This should rather go into TEST_DEPS.

**Comment by cockpituous** [comment]
image-refresh debian-stable done: https://github.com/cockpit-project/bots/commits/image-refresh-debian-stable-20230718-120426

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-stable-20230718-120404.log

**Comment by cockpituous** [comment]
image-refresh debian-testing done: https://github.com/cockpit-project/bots/commits/image-refresh-debian-testing-20230718-122719

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/debian-testing-20230718-122727.log

**Comment by cockpituous** [comment]
image-refresh ubuntu-stable done: https://github.com/cockpit-project/bots/commits/image-refresh-ubuntu-stable-20230718-124211

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/ubuntu-stable-20230718-124242.log

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/ubuntu-2204-20230718-125656.log

**Review by martinpitt — APPROVED** [review]
Nice, that went straight through!

**Pull request merged**